yt/frontends/flash/io.py

```diff
@@ -183,6 +183,12 @@ def _read_particle_coords(self, chunks, ptf):
                     total += count
                     yield ptype, (x, y, z)
 
+    def _yield_coordinates(self, data_file, needed_ptype=None):
+        px, py, pz = self._position_fields
+        p_fields = self._handle["/tracer particles"]
+        pxyz = np.asarray(p_fields[data_file.start:data_file.end], dtype="=f8")
+        yield ("io", pxyz)
```

Comment:
I don't think this does what you want--I think it grabs all of the fields instead of only the position ones.

Reply:
HA!  You're right.  Before my correction for flake8, it was `p_fields[data_file.start:data_file.end,(px,py,pz)]`.